src/build.py

```diff
@@ -1259,6 +1268,7 @@ def LibCXX():
   command = [PREBUILT_CMAKE_BIN, '-G', 'Ninja', os.path.join(LIBCXX_SRC_DIR),
              '-DCMAKE_CXX_COMPILER_WORKS=ON',
              '-DCMAKE_C_COMPILER_WORKS=ON',
+             '-DCMAKE_EXE_LINKER_FLAGS=-nostdlib++',
```

Comment:
I understand why this line should fix the current waterfall failures.
What is really don't understand is how it ever worked without this line?
I can repro this locally not BTW (the failure) while jacob can't?
Also, shouldn't this be the default for the libcxx build?   libcxx should not expect an existing libc++ library to exist on the system when it builds should it?

Reply:
Yeah I don't understand why it worked either. And yes you'd think that libcxx should not necessarily expect an existing stdlib; although on non-Apple platforms there always is one, e.g. gcc's libstdc++.

Reply:
(btw what version of CMake are you both using?)

Reply:
I build almost everything with `build.py`, so the prebuilt cmake (3.7.2)

Reply:
I'm using the waterfall one too.

Reply:
This line doesn't seem related to the rest of the change anymore, and the immediate issue is fixed.  Can we drop this line for now?